lib/cylc/review.py

```diff
@@ -263,6 +263,7 @@ def cycles(
             return self.template_env.get_template("cycles.html").render(**data)
         except jinja2.TemplateError:
             traceback.print_exc()
+        self._db_close(user, suite)
```

Comment:
Now that you've switched over to the simpler non-db way of telling Cylc 8 from 7 I think we might be best to revert these DB closes for now in-case they have unintended side effects.

Reply:
I totally agree, I've removed them now.